### ISSUE TITLE
[GUI-NCC] Phiếu cập nhật điều chỉnh thanh toán

### ISSUE THREAD

**Issue by dinhhung09138** [dinhhung09138/store-Adminstration#35]
- Xem file đính kèm
- Sử dụng bootstrap để thiết kế
- Không bị vỡ giao diện trên web, ipad và thiết bị di động
![ncc-cap-nhat-dieu-chinh-thanh-toan.PNG](https://zube.io/files/personal-5/12a374ff645d7fa7afdafac7aaddbe1f.png)

**Comment by dinhhung09138** [comment]
Chiến làm task này

**Comment by DoMinhChien** [comment]
<!DOCTYPE html>
<html>
<head>
	    
    <meta charset="utf-8">

    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
	<title>Phiếu Nhập</title>
	
	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta/css/bootstrap.min.css" integrity="sha384-/Y6pD6FV/Vv2HJnA6t+vslU6fwYXjCFtcEpHbNJ0lyAFsXTsjBbfaDjzALeQsN6M" crossorigin="anonymous">
	<link href="//netdna.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap.min.css" rel="stylesheet">
	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
</head>
<body>
	<div class="row">
		<div class="col-md-1"></div>
		<div class="col-md-2"><h3>Điều chỉnh</h3></div>
	</div>

	<div class="form-group row">
	<div class="col-md-1"></div>
  <label for="example-text-input" class="col-2 col-form-label">Mã Phiếu:</label>
  <div class="col-xs-6 col-sm-4">
    <input class="form-control" type="text" value="CB0000002" id="example-text-input">
  </div>
</div>
<div class="form-group row">
<div class="col-md-1"></div>
  <label for="example-search-input" class="col-2 col-form-label">Ngày điều chỉnh:</label>
  <div class="col-xs-6 col-sm-4">
    <input class="form-control" type="datetime-local" value="2011-08-19T13:45:00" id="example-datetime-local-input">
  </div>
  </div>
  <div class="form-group row">
<div class="col-sm-1"></div>
  <label for="example-search-input" class="col-2 col-form-label">Giá trị nợ điều chỉnh:</label>
  <div class="col-xs-6 col-sm-4">
    <input class="form-control" type="number" value="100">
  </div>
  </div>

    <div class="form-group row">
<div class="col-sm-1"></div>
  <label for="example-search-input" class="col-2 col-form-label">Người tạo:</label>
  <div class="col-xs-6 col-sm-4">
    <input class="form-control" type="title" value="Trần Đình Hưng">
  </div>
  </div>

   <div class="form-group row">
		<div class="col-sm-1"></div>
 		 <label for="example-search-input" class="col-2 col-form-label">Mô tả:</label>
	  <div class="col-xs-6 col-sm-4">
	   <textarea class="form-control" placeholder="Mô tả"></textarea>
	  </div>
  </div>

     <div class="form-group row">
		<div class="col-sm-1"></div>
		 <label for="example-search-input" class="col-2 col-form-label"></label>
	  <div class="col-xs-6 col-sm-4">
		<button type="submit" class="btn btn-success">
		<span class="glyphicon glyphicon-ok"></span>
		Cập nhật</button>
		<button type="submit" class="btn btn-secondary"><span class="glyphicon glyphicon-ban-circle"></span> Bỏ qua</button>
		<button type="submit" class="btn btn-danger"><span class="glyphicon glyphicon-remove"></span> Hủy bỏ</button>

	  </div>
  </div>
</body>
</html>

**Comment by DoMinhChien** [comment]
[task.txt](https://github.com/dinhhung09138/store-Adminstration/files/1220322/task.txt)

**Comment by DoMinhChien** [comment]
em làm xong task rồi a vào check giúp em nha!

**Comment by DoMinhChien** [comment]
[task.zip](https://github.com/dinhhung09138/store-Adminstration/files/1220325/task.zip)

**Comment by phonG11Hoai** [comment]
[Responsebility1.zip](https://github.com/dinhhung09138/store-Adminstration/files/1220404/Responsebility1.zip)

**Comment by dinhhung09138** [comment]
### Điểm đạt
- Sử dụng thẻ div, đúng yêu cầu.
- Sử dụng grid system của bootstrap 
- Sử dụng form-control

### Chưa đạt
- Khi chuyển qua giao diện tablet, mobile vỡ giao diện

### Cách sửa
- Đối với các thẻ điv khi chia cột, Sử dụng thêm **col-sm** và **col-xs** để áp dụng cho giao diện tablet và mobile để giao diện được như ý muốn


====> Em sửa lại và gởi lại giúp anh

**Comment by DoMinhChien** [comment]
[task.zip](https://github.com/dinhhung09138/store-Adminstration/files/1236194/task.zip)
